src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackLogExporter.cs

```diff
@@ -133,6 +133,11 @@ public ExportResult Export(in Batch<LogRecord> batch)
             }
         }
 
+        if (result == ExportResult.Success)
```

Comment:
This would depend on the last LogRecord in the batch. I think its best to simply emit - ExportCompleted. Success or Not is not required now. (If failed, then its already logged.)